app/views/vacancies/_vacancy.html.erb

```diff
@@ -1,11 +1,18 @@
-<section class="vacancy-item">
-  <div class="vacancy-title">
-    <%= link_to vacancy_path(vacancy) do %>
-      <%= vacancy.job_title %>
-    <% end %>
-  </div>
-  <div class="vacancy-details">
-    <span><i class="fa fa-briefcase" aria-hidden="true"></i><%= vacancy.company_name %></span>
-    <span><i class="fa fa-map-marker" aria-hidden="true"></i><%= vacancy.location %></span>
+<section class="vacancy-item grow">
+  <div class="row">
+    <div class="col-md-10">
+      <div class="vacancy-title">
+        <%= link_to vacancy_path(vacancy) do %>
```

Comment:
Since this line is not a multi tags text, change this to `<%= link_to vacancy.job_title, vacancy_path(vacancy) %>`